Solutions/Lookout Cloud Security/Data Connectors/LookoutCSConnector/LookoutCloudSecurityAnamolies/__init__.py

```diff
@@ -0,0 +1,248 @@
+import sys
+import azure.functions as func
+import datetime
+import json
+import base64
+import hashlib
+import hmac
+import requests
+import re
+import os
+import logging
+from azure.storage.fileshare import ShareClient
+from azure.storage.fileshare import ShareFileClient
+from azure.core.exceptions import ResourceNotFoundError
+
+jwt_api_key = os.environ['LookoutClientId']
+jwt_api_secret = os.environ['LookoutApiSecret']
+customer_id = os.environ['WorkspaceID']
+shared_key = os.environ['WorkspaceKey']
+connection_string = os.environ['AzureWebJobsStorage']
+logAnalyticsUri = os.environ.get('logAnalyticsUri')
+baseurl =  os.environ['Baseurl'] 
+maxResults = os.environ['MaxResults'] 
+Authurl = baseurl+"/apigw/v1/authenticate"
+table_name = "LookoutCloudSecurity"
+Schedule = os.environ['Schedule']
+fetchDelay = os.environ['FetchDelay']
+pastDays = os.environ['PastDays']
+chunksize = 500
+token = ""
+if(fetchDelay is None):
+ fetchDelay = 5
+
+if(pastDays is None):
+    pastDays = 7
```

Comment:
I think, if customer choose None, should we not process historical data?

Reply:
As discussed applied default values